README.md

```diff
@@ -240,43 +78,23 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/gjuliao/vet_clinic/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project please give us a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+I would like to thank the Microverse team for arranging the project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Actually When I click the link to the `MIT.md` file that you added, I get this 404 `error`. let's fix this 💯 👍🏻 

- [ ] I also recommend that you delete the Note: 👇🏻 in your License section as these are instructions on how to complete the License section.

![image](https://user-images.githubusercontent.com/79058364/220774566-5a904536-427b-4006-a1ae-d1cef45cfbe7.png)

Reply:
Yes, I notice the previous commit didn't included that part and already made it please check. @BenMukebo